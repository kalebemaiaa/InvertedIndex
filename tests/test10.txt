QueryAndS(QueryOr(QueryWord("ministro"), QueryWord("prefeito")), QueryWord("governador"))
3309.raw
11225.raw
1172.raw
5439.raw
151.raw
6227.raw
486.raw
5212.raw
4845.raw
2785.raw
6120.raw
5817.raw
3971.raw
2550.raw
11338.raw
5714.raw
2764.raw
10771.raw
18.raw
3817.raw
11587.raw
3209.raw
2571.raw
4833.raw
5398.raw
483.raw
1782.raw
4176.raw
2664.raw
5949.raw
4578.raw
8877.raw
2593.raw
11474.raw
4849.raw
1570.raw
3332.raw
10855.raw
4438.raw
6404.raw
89.raw
4000.raw
2270.raw
841.raw
4366.raw
1479.raw
4647.raw
11136.raw
4555.raw
596.raw
328.raw
796.raw
5041.raw
208.raw
4659.raw
2865.raw
11535.raw
11712.raw
11004.raw
9011.raw
10945.raw
2096.raw
3489.raw
11531.raw
11392.raw
1578.raw
6428.raw
3824.raw
11680.raw
272.raw
320.raw
259.raw
4816.raw
921.raw
5353.raw
5007.raw
901.raw
3844.raw
2945.raw
4197.raw
5347.raw
5230.raw
11822.raw
6414.raw
2913.raw
4700.raw
11405.raw
6030.raw
3464.raw
4408.raw
2864.raw
8868.raw
937.raw
1913.raw
10912.raw
4962.raw
2586.raw
1448.raw
3124.raw
3133.raw
93.raw
1588.raw
6192.raw
2527.raw
11407.raw
4929.raw
2562.raw
3876.raw
746.raw
4626.raw
2498.raw
11117.raw
2790.raw
1885.raw
1242.raw
11576.raw
919.raw
6138.raw
5961.raw
11349.raw
476.raw
1409.raw
4601.raw
4585.raw
4269.raw
10825.raw
11043.raw
5682.raw
2311.raw
1644.raw
11603.raw
5683.raw
4024.raw
1359.raw
11177.raw
6001.raw
552.raw
3949.raw
1373.raw
419.raw
1439.raw
588.raw
6298.raw
1564.raw
2274.raw
5589.raw
11770.raw
5696.raw
738.raw
4260.raw
1019.raw
1895.raw
6214.raw
11749.raw
11634.raw
1697.raw
5772.raw
11974.raw
11486.raw
4746.raw
6273.raw
10860.raw
11649.raw
10778.raw
2494.raw
5709.raw
5694.raw
1601.raw
4207.raw
2239.raw
10785.raw
11972.raw
1805.raw
11057.raw
1161.raw
5343.raw
2959.raw
3739.raw
11513.raw
5372.raw
11139.raw
3368.raw
3160.raw
5615.raw
2521.raw
3642.raw
4243.raw
1148.raw
3188.raw
3572.raw
148.raw
900.raw
4403.raw
10991.raw
5235.raw
5656.raw
791.raw
6278.raw
473.raw
3703.raw
