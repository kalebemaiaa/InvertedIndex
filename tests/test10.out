25
742
759
764
895
1067
1578
1597
1692
1962
1971
2079
2129
2243
2274
2349
2450
2496
2657
2667
3055
3224
3244
3256
3526
3557
3775
3778
3796
4448
4491
4523
4581
4637
4893
4990
5273
5380
5449
5775
5814
6066
11175
11461
11524
11646
11911
11979
